### PR TITLE
Note about transactional update timer

### PR DESCRIPTION
**Pull request by admd**
# Description

- Note about transactional update timer.
- Fixes https://bugzilla.suse.com/show_bug.cgi?id=1226843

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 5.0). 

Backport targets (edit as needed):

- master
- 5.0

# Links
- This PR tracks issue https://github.com/SUSE/spacewalk/issues/24631

### CODE REVIEW

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc [hunk 1]
```diff
@@ -245,11 +245,11 @@ You can consider generated certificates as {productname} builtin (self signed) c
 +
 Depending on the choice then provide either path to signing CA certificate to generate a new certificate or path to an existing certificate and its key to be used as proxy certificate.
 +
-The CA certificates generated on the server are stored in the [path]``/var/lib/containers/storage/volumes/root/ssl-build`` directory.
+The CA certificates generated on the server are stored in the [path]``/var/lib/containers/storage/volumes/root/_data/ssl-build`` directory.
 +
 For more information about existing or custom certificates and the concept of corporate and intermediate certificates, see  xref:administration:ssl-certs-imported.adoc[].
 
-. Click btn:[Generate] to register new proxy FQDN in {productname} Server and generate configuration archive with details for container host.
+. Click btn:[Generate] to register new proxy FQDN in {productname} Server and generate configuration archive (config.tar.gz) with details for container host.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc:252]:
```suggestion
. Click btn:[Generate] to register new proxy FQDN in {productname} Server and generate configuration archive ([filename]``config.tar.gz``) with details for container host.
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc [hunk 2]
```diff
@@ -263,13 +263,8 @@ This archive needs to be made available on the proxy container host.
...
 
 .Task: Copy the Proxy Configuration
-. Copy the files from the Server container to the Server host OS:
-+
-----
-mgrctl cp server:/root/config.tar.gz .
-----
 
-. Next copy the files from the Server host OS to the Proxy host: 
+. Copy the locally downloaded configuration archive generated in the previous step to the Proxy host.:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc:267]:
```suggestion
. Copy the locally downloaded configuration archive generated in the previous step to the Proxy host:
```

**Review by srbarrios — APPROVED** [review]
Very much needed! Thank you for that warning!

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc [hunk 3]
```diff
@@ -245,11 +245,11 @@ You can consider generated certificates as {productname} builtin (self signed) c
 +
 Depending on the choice then provide either path to signing CA certificate to generate a new certificate or path to an existing certificate and its key to be used as proxy certificate.
 +
-The CA certificates generated on the server are stored in the [path]``/var/lib/containers/storage/volumes/root/ssl-build`` directory.
+The CA certificates generated on the server are stored in the [path]``/var/lib/containers/storage/volumes/root/_data/ssl-build`` directory.
```
Comment (jcayouette) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc:248]: generated by the server

File: modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc [hunk 4]
```diff
@@ -236,6 +236,22 @@ than 48 characters in length.
 . Open a browser and proceed to your servers FQDN.
 . Enter your username (default is [literal]``admin``) and the password you set during the deployment process.
 
+
```
Comment (jcayouette) [inline modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc:239]: I would reword this section:
[IMPORTANT]
SLE Micro is designed to update itself automatically by default and will reboot after applying updates. However, this behavior is not desirable for the {productname} environment. To prevent automatic updates on your server, we recommend disabling the transactional-update timer.

You can disable the timer by running the following command:

[source, shell]
----
systemctl disable --now transactional-update.timer
----

**Review by jcayouette — APPROVED** [review]
